tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -660,6 +660,45 @@ describe('metrics selectors', () => {
     });
   });
 
+  describe('getMetricsCardDataMinMax', () => {
+    it('returns undefined when cardStateMap is undefined', () => {
+      const state = appStateFromMetricsState(buildMetricsState({}));
+      expect(
+        selectors.getMetricsCardDataMinMax(state, 'card1')
+      ).toBeUndefined();
+    });
+
+    it('returns undefined when card has no cardState', () => {
+      const state = appStateFromMetricsState(
+        buildMetricsState({
+          cardStateMap: {},
+        })
+      );
+      expect(
+        selectors.getMetricsCardDataMinMax(state, 'card1')
+      ).toBeUndefined();
+    });
+
```

Comment:
nit: Can we have cardStateMap and the card id but the dataMinMax is undefined?

Reply:
That should not be possible but I can add a test.